components/automate-ui/src/app/pages/project/list/project-list.component.spec.ts

```diff
@@ -251,203 +237,12 @@ describe('ProjectListComponent', () => {
     });
   });
 
-  describe('when update-start-confirmation modal emits a cancellation', () => {
```

Comment:
Rather than just deleting these tests, could you relocate them into the pending/progress bar components?

Reply:
@msorens most of the tests were coded to check for a button text change. I'll work on recreating the tests with the new changes.

Reply:
@msorens updated tests👍🏻